depenguinme.sh

```diff
@@ -164,6 +164,8 @@ done
 
 if [ "$REQUIRE_SSHKEY" = "YES" ]; then
 	[ -s COPYKEY.pub ] || exit_error "Authorized key sources are empty"
+else
```

Comment:
The command in line 155 already creates an empty file, so this isn’t necessary

Reply:
Maybe go explain, “:” means “true” and >filename overwrites filename, so `: >COPYFILE.pub` makes sure an empty file of that name exists (regardless if it existed beforehand or not)